index.mjs

```diff
@@ -24,7 +24,7 @@ import chalk from 'chalk'
 import minimist from 'minimist'
 
 export function $(pieces, ...args) {
-  let __from = (new Error().stack.split('at ')[2]).trim()
+  let __from = (new Error().stack.split(/^\s*at\s/m)[2]).trim()
```

Comment:
Maybe just `.split(' at ')`?

Reply:
Yes, that would also fix the issue of function names ending with 'at'. 

As improbable as it may be though, consider the following code:
```js
function test0() {
  f();
}

function f() {
  console.log(new Error().stack);
  console.log("simple", new Error().stack.split(' at ')[2]);
  console.log("complex", new Error().stack.split(/^\s*at /m)[2]);
}

test0();
```

With `.split(' at ')` this would fail if the file name contained spaces and the word at (e.g. `'file at home`).

I think that a robust solution would not involve parsing `new Error().stack` though.

Reply:
Got it.

I think it's not possible to not use `new Error().stack`. I don't know another way to get callee name and path.